RELEASE/scripts/autoscend/auto_powerlevel.ash

```diff
@@ -394,6 +394,12 @@ boolean LX_freeCombats(boolean powerlevel)
 		if(adv_done) return true;
 	}
 
+	/*if(item_amount($item[Map to a candy-rich block]) > 0 && !get_property("_mapToACandyRichBlockUsed").to_boolean() && count(get_outfits()) > 3)
```

Comment:
either remove or uncomment; don't add commented out code

Reply:
Simplified and uncommented. Should be fine to powerlevel (or other free fight uses) using trick-or-treat now.